index.md

```diff
@@ -15,17 +15,24 @@ For more general information about the Managed JupyterHub Service, see [the Team
 
 See the sections below (or to the left) for more information.
 
-## About the 2i2c hubs
+## About the Managed JupyterHubs Pilot
+
+The Managed JupyterHubs Pilot is an experiment at building an open, scalable, sustainable cloud service for interactive computing environments in research and education.
```

Comment:
Do you think `experiment` is the proper word here? For me, it is actually transmitting an unstable concept I do not think it is actually represented by our deployments. WDYT about "our attempt to"?
(Forget this comment if this is just me because English is not my mother language).

Reply:
good point, I agree 👍